Makefile

```diff
@@ -1,6 +1,31 @@
+current_dir = $(shell pwd)
+
+.PHONY: check
 check:
 	flake8 --config .flake8-code .
 	flake8 --config .flake8-doc .
 	pylint lookout
 
-.PHONY: check
+.PHONY: docker-build
+docker-build: Dockerfile
+	docker build -t srcd/style-analyzer .
+
+.PHONY: docker-test
+docker-test:
+	$(MAKE) _docker-test || docker rm -f style_analyzer_bblfshd
+
+.PHONY: _docker-test
+_docker-test:
+	docker run -d --rm --name style_analyzer_bblfshd \
```

Comment:
It is either `-d` or `--rm` but not together.

Reply:
Actually, it is possible since https://github.com/moby/moby/pull/20848 but we do not need `--rm` here. I can remove it.

Reply:
Oh nice they finally added this feature. Indeed since we manually stop it `--rm` is not needed.